app/controllers/api/v1/media_controller.rb

```diff
@@ -31,7 +31,7 @@ def status_code_for_media_attachment
   end
 
   def set_media_attachment
-    @media_attachment = current_account.media_attachments.unattached.find(params[:id])
+    @media_attachment = current_account.media_attachments.find(params[:id])
```

Comment:
Plus this may allow a single media attachment to be attached to multiple statuses.

Reply:
How?

Reply:
`UpdateStatusService` only allows attaching media that is either unattached or was already attached to the status being edited. It is not possible to yoink a media attachment from another status.